sefaria/helper/crm/crm_mediator.py

```diff
@@ -20,8 +20,8 @@ def create_crm_user(self, email, first_name, last_name, lang="en", educator=Fals
         except:
             return False
 
-    def subscribe_to_lists(self, email, first_name, last_name, educator=False, lang="en"):
-        return self._crm_connection.subscribe_to_lists(email, first_name, last_name, educator, lang)
+    def subscribe_to_lists(self, email, first_name, last_name, educator=False, lang="en", mailing_lists=[]):
```

Comment:
Don't use empty list as a default parameter in Python. This can cause strange bugs. The standard is to use `None`.

Reply:
Yeah, thanks, good catch 🙂 I realized that in the other spot of my code that Python evaluates default arguments when the function is first defined and not when the function is called again other times.